frontend/test/__runner__/cypress-runner-backend.js

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env node
+
+const fs = require("fs");
+const os = require("os");
+const path = require("path");
+const { spawn } = require("child_process");
+
+const fetch = require("isomorphic-fetch");
+
+const CypressBackend = e2eBackend();
+
+function e2eBackend() {
```

Comment:
I think we can remove a layer here and just have `CypressBackend` set to what `e2eBackend()`, without actually calling `e2eBackend`
```suggestion
const CypressBackend = {
 createServer() {...},
 start(server) {...},
 ...
}
```

Reply:
Haha, @gusaiani beat me to it

Reply:
@npfitz done! Thank you for the suggestion.